.travis.yml

```diff
@@ -1,8 +1,12 @@
 language: java
 jdk:
   - oraclejdk8
-script: mvn clean verify
+script: "mvn cobertura:cobertura"
```

Comment:
Does this command include the `mvn verify` phase? I think findbugs is run during the mvn verify phase.